pkg/email/patch.go

```diff
@@ -33,7 +33,14 @@ func ParsePatch(message []byte) (diff string) {
 			}
 		}
 	}
-	if err := s.Err(); err != nil {
+	err := s.Err()
+	if err == bufio.ErrTooLong {
+		// It's a problem of the incoming patch, rather than anything else.
```

Comment:
Btw, scanner supports setting max line length if needed. But AFAIR the default is 4K, there should be no such long lines in well-formed kernel emails.

Reply:
> there should be no such long lines in well-formed kernel emails

There definitely were a few exceptions. Anyway, it's better not to panic on user input problems :)